.github/CODEOWNERS

```diff
@@ -52,6 +52,7 @@
 /jobs/**/*                             @GoogleCloudPlatform/python-samples-reviewers
 /kubernetes_engine/**/*                @GoogleCloudPlatform/python-samples-reviewers
 /kubernetes_engine/django_tutorial/**/*    @glasnt @GoogleCloudPlatform/python-samples-reviewers
+/language/**/*                         @googleapis/python-samples-reviewers @googleapis/cdpe-cloudai
```

Comment:
Could you find a team from `GoogleCloudPlatform`? Primary team (product team like cdpe/cloud-ai or equivalent) should come first, followed by the default reviewers (python-sample-reviewers).